docs/Lets-Write-a-Test.md

```diff
@@ -38,36 +42,38 @@ Here's our magic command that restarts mcollective:
 ####Check to see if more than one mcollective process is running
 
 Here's our magic command that throws an error if more than one mcollective process is running:
-```
-process_count_check = "bash -c '[[ $(ps auxww | grep [m]collectived | wc -l) -eq 1 ]]'"
-```
+
+    process_count_check = "bash -c '[[ $(ps auxww | grep [m]collectived | wc -l) -eq 1 ]]'"
+
 ###Put it all together
 
 Here's the finished acceptance test.
-```
-test_name "/etc/init.d/pe-mcollective restart check"
-
-# Don't run these tests on the following platforms
-confine :except, :platform => 'solaris'
-confine :except, :platform => 'windows'
-confine :except, :platform => 'aix'
-
-step "Make sure the service restarts properly"
-hosts.each do |host|
-  # Commands to execute on the target system.
-  restart_command = "bash -c '[[ -x /etc/init.d/pe-mcollective ]] && /etc/init.d/pe-mcollective restart'"
-  process_count_check = "bash -c '[[ $(ps auxww | grep [m]collectived | wc -l) -eq 1 ]]'"
-
-  # Restart once
-  on(host, restart_command) { assert_equal(0, exit_code) }
-  # Restart again
-  on(host, restart_command) { assert_equal(0, exit_code) }
-  # Check to make sure only one process is running
-  on(host, process_count_check) { assert_equal(0, exit_code) }
-end
-```
+
+    test_name "/etc/init.d/pe-mcollective restart check"
```

Comment:
there's one advantage to using the three backticks method for this code block, and it's that you get ruby-specific highlighting, which I find really nice.  You can get that here by specifying

    ```ruby

as the opening mark on the code block. I thought about mentioning it above, but that line won't get as much from doing something like this since it's really ruby quoting a bash command, but here it could be nice for readers.